docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -92,63 +95,167 @@ You can also build the collector-contrib repository by cloning it and running:
 make otelcontribcol
 ----
```

Comment:
@axw @EamonnTP I propose to remove the explanation on how to build yourself the collector. The collector is stable enough and official releases cover the feature we need.
Building your own Otel Collector is now an advanced use case, it's not the purpose of this documentation page.

Reply:
Thanks, will do.